akka-projection-testkit/src/test/java/akka/projection/testkit/javadsl/ProjectionTestKitTest.java

```diff
@@ -59,39 +65,27 @@ public void assertProgressOfAProjection() {
     StringBuffer strBuffer = new StringBuffer();
     TestProjection prj = new TestProjection(src, strBuffer, i -> i <= 6);
 
-    projectionTestKit.run(prj, () -> assertEquals(strBuffer.toString(), "1-2-3-4-5-6"));
+    projectionTestKit.run(prj, () -> assertEquals("1-2-3-4-5-6", strBuffer.toString()));
   }
 
   @Test
   public void retryAssertionFunctionUntilItSucceedsWithinAMaxTimeout() {
 
     StringBuffer strBuffer = new StringBuffer();
-
-    Source<Integer, NotUsed> delayedSrc =
-        src.delayWith(
-            () -> DelayStrategy.linearIncreasingDelay(Duration.ofMillis(200), __ -> true),
-            DelayOverflowStrategy.backpressure());
-
-    TestProjection prj = new TestProjection(delayedSrc, strBuffer, i -> i <= 6);
-
+    TestProjection prj = new TestProjection(delayedSrc(100), strBuffer, i -> i <= 6);
```

Comment:
The delay was 200, now is 100. In total, it should take 600ms to start failing.

The failure in #285 shows that after 2 seconds not all envelopes were processed. That's odd.

Reply:
Just out of curiosity: how long does it take to run the test locally or in TravisCI? do we have the times? Hmmm, this is JUnit, not scalatest so not sure we have the metrics...

Reply:
yes, indeed we don't have the timings